app/src/main/java/com/kickstarter/ui/activities/ThreadActivity.kt

```diff
@@ -78,6 +78,16 @@ class ThreadActivity :
                 this.repliesAdapter.takeData(it.first.reversed())
             }
 
+        this.viewModel.outputs
+            .onCommentReplies()
+            .compose(bindToLifecycle())
+            .observeOn(AndroidSchedulers.mainThread())
+            .filter { it.first.isNullOrEmpty() }
```

Comment:
Can the filter be applied on the ViewModel?

Reply:
@Arkariang  
I changed 
![image](https://user-images.githubusercontent.com/1075310/126194795-6f5cc5d4-712b-4f51-a349-50f5592df091.png)